imaginary.go

```diff
@@ -241,7 +241,7 @@ func parseOrigins(origins string) []*url.URL {
 func memoryRelease(interval int) {
 	ticker := time.NewTicker(time.Duration(interval) * time.Second)
 	go func() {
-		for _ = range ticker.C {
+		for range ticker.C {
```

Comment:
This is not a valid syntax in Go 1.3 or lower versions. I'm open to officially Go 1.5+ from now. 
Can you update `.travis.yml`?

See: https://travis-ci.org/h2non/imaginary/jobs/236177548#L3727-L3728